lib/common/containers/ActiveSidebarNavItem.js

```diff
@@ -21,8 +21,8 @@ const mapStateToProps = (state: AppState, ownProps: Props) => {
 }
 
 const mapDispatchToProps = {}
-
-var ActiveSidebarNavItem = connect(
+// $FlowFixMe https://github.com/flow-typed/flow-typed/issues/2628
```

Comment:
Trying to resolve this flow error is hella annoying. Guess I have to let it slide.